src/linter/conf.go

```diff
@@ -33,28 +33,36 @@ var (
 	// SrcInput implements source code reading from files and buffers.
 	//
 	// TODO(quasilyte): avoid having it as a global variable?
```

Comment:
Probably, the comment about globality can be removed.

Reply:
Fixed.